src/pip/_internal/network/lazy_wheel.py

```diff
@@ -1,66 +1,122 @@
 """Lazy ZIP over HTTP"""
 
-__all__ = ["HTTPRangeRequestUnsupported", "dist_from_wheel_url"]
+from __future__ import annotations
 
+__all__ = ["HTTPRangeRequestUnsupported", "dist_from_wheel_url", "LazyWheelOverHTTP"]
```

Comment:
I think we tend to put this after the imports

Reply:
Done!